tools/please_pex/pex/pex.go

```diff
@@ -46,20 +46,44 @@ func (pw *Writer) SetShebang(shebang string) {
 
 // SetTest sets this Writer to write tests using the given sources.
 // This overrides the entry point given earlier.
-func (pw *Writer) SetTest(srcs []string, usePyTest bool) {
+func (pw *Writer) SetTest(srcs []string, testRunner string) {
 	pw.realEntryPoint = "test_main"
 	pw.testSrcs = srcs
-	if usePyTest {
+
+	commonIncludes := []string{
+		".bootstrap/coverage",
+		".bootstrap/__init__.py",
+		".bootstrap/six.py",
+	}
+
+	if testRunner == "pytest" {
 		// We only need xmlrunner for unittest, the equivalent is builtin to pytest.
-		pw.testExcludes = []string{".bootstrap/xmlrunner/*"}
```

Comment:
I think you can probably delete the `testExcludes` member entirely now, I don't think anything will use it any more.